application/src/main/java/org/opentripplanner/routing/algorithm/filterchain/ItineraryListFilterChainBuilder.java

```diff
@@ -89,6 +91,7 @@ public class ItineraryListFilterChainBuilder {
   private double minBikeParkingDistance;
   private boolean removeTransitIfWalkingIsBetter = true;
   private ItinerarySortKey itineraryPageCut;
+  private OptionalInt streetOnlyCost = OptionalInt.empty();
```

Comment:
Optional field here again.

Reply:
I created a `RemoveTransitIfStreetOnlyIsBetterResults` record and removed optionals from fields